resources/js/Components/Collections/CollectionActivityTable/CollectionActivityTable.blocks.tsx

```diff
@@ -17,6 +17,8 @@ import { formatAddress } from "@/Utils/format-address";
 import { isTruthy } from "@/Utils/is-truthy";
 import { TruncateMiddle } from "@/Utils/TruncateMiddle";
 
+const commonText = "common.from";
```

Comment:
why was this needed? feels like an eslint rule that we're better off ignoring in this case 🤔